packages/core/test/tracing/index_test.js

```diff
@@ -100,9 +117,10 @@ mochaSuiteFn('[UNIT] tracing/index', function () {
   });
 
   it('skip init step when instrumentation disabled', () => {
-    initAndActivate({ tracing: { disabledTracers: ['grpcjs', 'kafkajs'] } });
+    initAndActivate({ tracing: { disabledTracers: ['grpcjs', 'kafkajs', 'aws-sdk/v2'] } });
     expect(initStubKafkaJs).not.to.have.been.called;
     expect(initStubGrpcJs).not.to.have.been.called;
+    expect(initAwsSdkv2).to.not.have.been.called;
     expect(initStubGrpc).to.have.been.called;
   });
```

Comment:
_though:_ We could merge this into 'deactivate instrumentation via config' instead of having a separate test case, see above.